HW_7/34.py

```diff
@@ -0,0 +1,18 @@
+def rhyme(txt):
+    ph = str.lower(txt).split()
```

Comment:
Можно и по другому, без обращения к классу
```suggestion
    ph = txt.lower().split()
```